code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm

```diff
@@ -196,17 +203,24 @@
 
 	var/turf/T = X.loc
 	var/turf/temp = X.loc
+	var/empowered = X.empower() //Should it knockdown everyone down its path ?
```

Comment:
useless var, just call X.empower() directly?

Reply:
Can't exactly do that since the check for empowered happens in a loop, calling it on each one would give lots of combo.

Reply:
hum right